Dispatchr/app/actions/types.js

```diff
@@ -1,3 +1,6 @@
 export const ADD_NEW_REQUEST = 'ADD_NEW_REQUEST';
 export const REQUEST_REQUESTED = 'REQUEST_REQUESTED';
 export const REQUEST_RECEIVED = 'REQUEST_RECEIVED';
+export const ADD_ITEM = 'ADD_ITEM';
```

Comment:
We are! I just didn't prioritize it right now. If you really want it, I can switch it to one actions file.